test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -100,13 +104,22 @@ namespace iroha {
       }
 
       /**
-       * Check that command result contains specific error code
+       * Check that command result contains specific error code and error
+       * message
        * @param cmd_result to be checked
        * @param expected_code to be in the result
+       * @param expected_substrings - collection of strings, which are expected
+       * to be in command error
        */
-#define CHECK_ERROR_CODE(cmd_result, expected_code) \
-  ASSERT_TRUE(err(cmd_result));                     \
-  ASSERT_EQ(err(cmd_result)->error.error_code, expected_code);
+#define CHECK_ERROR_CODE_AND_MESSAGE(                \
+    cmd_result, expected_code, expected_substrings)  \
+  auto error = err(cmd_result);                      \
+  ASSERT_TRUE(error);                                \
+  ASSERT_EQ(error->error.error_code, expected_code); \
```

Comment:
Also please after ASSERT_TRUE, change all asserts to expects, so that all checks are run in case some fail.

Check out this section of google test documentation: https://github.com/abseil/googletest/blob/master/googletest/docs/primer.md#assertions